tests/functional_tests/training/test_nvrx_straggler.py

```
@@ -238,6 +238,7 @@ def setup_test_logging(log_file: str, rank: int):
     return file_handler
 
 
+@pytest.mark.pleasefixme
```

Flake8 (7.3.0)

[error] undefined name 'pytest'

(F821)